tests/test-class-amp-theme-support.php

```diff
@@ -220,7 +220,6 @@ public function test_prepare_response() {
 	 * @covers AMP_Theme_Support::prepare_response()
 	 */
 	public function test_validate_non_amp_theme() {
-		global $wp_widget_factory, $wp_scripts;
 		$wp_scripts = null;
```

Comment:
This line still needs to be removed and this PR will be good to go. Thanks @amedina, great stuff!